substratevm/JFR.md

```diff
@@ -0,0 +1,79 @@
+This page covers how to use JDK Flight Recorder (JFR) with Java native images
```

Comment:
You can move the file to `graal/docs/reference-manual/native-image` (you will need to add a file header though, see the other files)

Reply:
@jiekang, I'll take care of adding the header files and making sure this doc is reachable from the website menu.

Reply:
@olyagpl Thank you. Should I still move this file to the directory `graal/docs/reference-manual/native-image` ?

Reply:
@jiekang, I'm about to push my edits to your branch if you don't mind.

Reply:
@olyagpl oh, cool; I didn't consider that option. That works for me. Thank you again! :)

Reply:
@jiekang, @christianhaeubl, please see my version of the file https://github.com/oracle/graal/pull/3549/commits/0f9e0cd749d40c0e4730e93856b312a3cc6546f6. I reshaped the content a bit to match other Native Image documentation.